image-patching/stopgap-script/nuage_overcloud_full_patch.py

```diff
@@ -146,161 +138,133 @@ def rhel_subscription(username, password, pool, image, proxy_hostname=None, prox
 # Function to remove the RHEL subscription
 #####
 
+
 def rhel_remove_subscription(image):
-    cmds_run(['cat <<EOT > rhel_unsubscribe \n'
-              'subscription-manager unregister \n'
-              'EOT'])
+    cmds_run([
+        '''cat <<EOT > rhel_unsubscribe
+#!/bin/bash
+subscription-manager unregister
+EOT'''
+    ])
     virt_customize_run(
-        'rhel_unsubscribe -a %s --memsize %s --selinux-relabel' % (
-        image, VIRT_CUSTOMIZE_MEMSIZE))
+        'rhel_unsubscribe -a %s --memsize %s --selinux-relabel' %
+        (image, VIRT_CUSTOMIZE_MEMSIZE))
     cmds_run(['rm -f rhel_unsubscribe'])
 
 
 #####
 # Function to remove packages that are not needed
 #####
 
+
 def uninstall_packages(image):
-    virt_customize(
-        '"yum remove openvswitch -y" -a %s --memsize %s '
-        '--selinux-relabel' % (
-        image, VIRT_CUSTOMIZE_MEMSIZE))
+    virt_customize('"yum remove openvswitch -y" -a %s --memsize %s '
+                   '--selinux-relabel' % (image, VIRT_CUSTOMIZE_MEMSIZE))
 
 
 #####
 # Function to install Nuage packages that are required
 #####
 
-def install_packages(image):
+
+def install_nuage_packages(image, nuage_vrs_repos):
+    enable_repos_cmd = "subscription-manager repos"
```

Comment:
`subscription-manager repos` can be only used for redhat repos but not custom repos.
For custom repos, we need to use `yum-config-manager`
```
[root@rhdirector avrs]# subscription-manager repos --enable=nuage_openstack
1 local certificate has been deleted.
Error: 'nuage_openstack' does not match a valid repository ID. Use "subscription-manager repos --list" to see valid repositories.
```

when I ran `yum-config-manager --enable nuage_openstack`, this repo got enabled